src/Approvals/SFA.DAS.Approvals.Api/Controllers/DraftApprenticeshipController.cs

```diff
@@ -75,6 +76,30 @@ public async Task<IActionResult> GetEditDraftApprenticeship(long cohortId, long
             }
         }
 
+        [HttpGet]
+        [Route("employer/{accountId}/unapproved/{cohortId}/apprentices/{draftApprenticeshipId}/view")]
+        [Route("provider/{providerId}/unapproved/{cohortId}/apprentices/{draftApprenticeshipId}/view")]
+        public async Task<IActionResult> GetViewDraftApprenticeship(long cohortId, long draftApprenticeshipId)
+        {
+            try
+            {
+                var result = await _mediator.Send(new GetViewDraftApprenticeshipQuery { CohortId = cohortId, DraftApprenticeshipId = draftApprenticeshipId });
+
+                if (result == null)
+                {
+                    return NotFound();
+                }
+
+                return Ok(result);
```

Comment:
We're not mapping to a response here and just returning the query result.

Reply:
ok, will map it thx

Reply:
fixed